test/test_jit.py

```diff
@@ -4732,6 +4732,43 @@ class TestJitGenerated(TestCase):
     pass
 
 
+UBSAN_BLACKLISTED_TESTS = [
```

Comment:
Why are those blacklisted? Do we know what’s wrong?

Reply:
This needs to be put as a comment in the code, but my understanding is that UBSAN per-function exclusions don't seem to work with OpenMP pragmas, maybe because internally OpenMP generates some extra functions which then don't get blacklisted. (The test themselves exhibit UB whenever they divide by zero.)